### PR TITLE
Biochem additions

### PR DESCRIPTION
**Pull request by JamesJeffryes**
2 new search queries plus some additional relationships to play with

### CODE REVIEW

**Comment by jayrbolton** [comment]
"float" should be "number" in json-schema speak

File: schemas/edges/rxn/rxn_reaction_linked_to_reaction.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (jayrbolton) [inline schemas/edges/rxn/rxn_reaction_linked_to_reaction.json]: What do you mean by "linked_to"? Maybe you can clarify in the description here.

File: schemas/vertices/rxn/rxn_compound [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      "type": ["string", "null"],
+      "description": "If the compound is deprecated, the compound that supersedes this entry"
+    },
+    "mass": {
+      "type": ["number", "null"],
+    },
+    "name": {
+      "type": "string",
+    },
+    "pka": {
```
Comment (jayrbolton) [inline schemas/vertices/rxn/rxn_compound]: Can you add some titles or descriptions on these fields (pka, pkb, smiles) to say what they are?

File: views/search_compounds.aql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+\\ Use ArangoSearch to search documents in the compounds index. Returns documents in their natural order
+\\ include_obsolete - should obsolete documents be included
+\\ offset - how many documents to skip
+\\ result_limit - Maximum documents to return
+FOR doc IN Compounds
+  SEARCH ANALYZER(PHRASE(doc.id, @search_text)
```
Comment (jayrbolton) [inline views/search_compounds.aql]: cool!!

**Comment by jayrbolton** [comment]
(done with my feedback now)

File: schemas/edges/rxn/rxn_reaction_linked_to_reaction.json (hunk 1 above)
Comment (JamesJeffryes) [reply schemas/edges/rxn/rxn_reaction_linked_to_reaction.json]: I'm a little leery of being more specific here than the source database but I'll add something

Comment (jayrbolton) [reply schemas/edges/rxn/rxn_reaction_linked_to_reaction.json]: Is the linkage some kind of cause and effect, even indirect? Or is it some other relation like similarity?

**Pull request merged**